### PR TITLE
Campaign API route: always redirect to user defined URL

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Always redirect to the user defined path after attaching campaign code.

## Justify why they are needed

Before we used the fallback path if something went wrong but this is unexpected behaviour.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1863** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1863" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1863?utm_source=stack-comment).

File: apps/store/src/pages/api/campaign/[code].ts [hunk 1]
```diff
@@ -38,9 +38,8 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
   nextUrl.searchParams.delete(QueryParam.Next)
   nextUrl.pathname = nextQueryParam
 
-  const locale = getUrlLocale(nextUrl.toString())
-  fallbackRedirect[1] = PageLink.home({ locale })
-
+  const redirect = [307, nextUrl.toString()] as const
```
Comment (alebedev) [inline apps/store/src/pages/api/campaign/[code].ts:41]: Having redirectStatus and redirectUrl as separate vars will probably make it easier to read

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/pages/api/campaign/[code].ts (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/pages/api/campaign/[code].ts:41]: When are we getting proper tuples in TS/JS??? 😭

**Pull request merged**